lib/Bort.pm

```diff
@@ -144,7 +144,9 @@ sub run_channel_watches {
       (ref $watch->[0] eq 'Regexp' && $text =~ m/$watch->[0]/) ||
       (ref $watch->[0] eq 'CODE' && $watch->[0]->($text));
     try {
-      $watch->[1]->($ctx, $text);
+      if (ref $watch->[1] eq 'CODE') {
```

Comment:
What's this about? Why would there ever not be a callback?

If we want to make sure you the caller always actually sets a callback, then we should do that in an args check in `register_channel_watch` and friends.

Reply:
Discussion: not here, not now.